packages/remirror__extension-entity-reference/src/entity-reference-extension.ts

```diff
@@ -169,6 +169,34 @@ export class EntityReferenceExtension extends MarkExtension<EntityReferenceOptio
     };
   }
 
+  /**
+   * Dispatch a transaction that selects the range of the entity reference then scrolls to it.
+   *
+   * @param entityReferenceId - The entity's reference Id.
+   *
+   * @returns True if the scrolling was applied, else it returns false
+   *
+   */
+  @command()
+  scrollToEntityReference(entityReferenceId: string): CommandFunction {
+    return ({ tr, dispatch }) => {
+      const entityReference = this.store.helpers.getEntityReferenceById(entityReferenceId);
```

Comment:
Minor, but I think you could 

```suggestion
      const entityReference = this.getEntityReferenceById(entityReferenceId);
```